README.md

```diff
@@ -112,25 +116,25 @@ of these methods accept planner-specific keyword arguments.
 ### Writing a Custom Planner
 
 Implementing a custom planner requires extending the `BasePlanner` class and
-decorating one or more methods with the `@PlanningMethod` decorator. Extending
-the `BasePlanner` class constructs the planning environment `self.env` and
-allows PrPy to identify your planner as a base planner class, as opposed to a
-meta-planner. The `@PlanningMethod` decorator handles environment cloning and
-allows meta-planners to query the list of planning methods that the planner
-supports (e.g. to generate docstrings).
+decorating one or more methods with the `@LockedPlanningMethod` or
+`@ClonedPlanningMethod` decorator. Extending the `BasePlanner` class constructs
+the planning environment `self.env` and allows PrPy to identify your planner
+as a base planner class, as opposed to a meta-planner. The `@*PlanningMethod`
+decorators handle environment cloning or locking and allows meta-planners to
+query the list of planning methods that the planner supports (e.g. to generate
+docstrings).
 
 Please obey the following guidelines:
 
-- Assume that the cloned environment is locked during the entire call.
-- Subclass constructor **must** call `BasePlanner.__init__`.
-- A `@PlanningMethod` **must not** call another `@PlanningMethod`.
-- Each `@PlanningMethod` **must** accept the first argument `robot`, which is a
+- Assume that the environment is locked during the entire call.
+- Subclass constructors **must** call `BasePlanner.__init__`.
+- Each `@*PlanningMethod` **must** accept the first argument `robot`, which is a
   robot in the cloned environment.
```

Comment:
This is no longer true. The environment will not be cloned when using `@LockedPlanningMethod`.